oaknational/Oak-Web-Application
# feat: json ld for webinars

## Description

- Refactors Json ld for blogs and webinars (list and slug page)

## Issue(s)

Fixes #936 

## How to test

1.Check out branch locally 
2.Use [ngrok](https://ngrok.com/product)
3.ngrok http 3000
4. https://search.google.com/test/rich-results and https://validator.schema.org/ to test for valid jsonld on blog listing and webinar pages 

webinars list page
![image](https://user-images.githubusercontent.com/7987787/200855863-b229311b-06b0-47b7-b776-fe30904eb31a.png)

webinars slug page
![image](https://user-images.githubusercontent.com/7987787/200856187-aa0db09c-a48b-4a29-a65a-0a31ba40233f.png)



1. Go to {cloud link}
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
